docs/api/security/crypto.md

```diff
@@ -0,0 +1,53 @@
+## Mbed Crypto
+
+Arm Mbed Crypto is the reference implementation of the cryptography interface
+of the Arm Platform Security Architecture (PSA).
+
+<span class="notes">**Note:** The version of Mbed Crypto shipping with Mbed OS
+implements PSA Crypto API v1.0b1.</span>
+
+We have adapted and [integrated Mbed Crypto with Mbed
+OS](https://github.com/ARMmbed/mbed-os/blob/master/features/mbedtls/mbedcrypto).
```

Comment:
I think this link should be: https://github.com/ARMmbed/mbed-os/tree/master/features/mbedtls/mbed-crypto